frontend/app/src/premium/register-components.ts

```diff
@@ -134,6 +135,8 @@ export function registerComponents(): void {
   // Version 24 - 1.31
   Vue.component('LpPoolHeader', LpPoolHeader);
   Vue.component('RowAppend', RowAppend);
+  // Version 25 - 1.32
+  Vue.component('UniswapPoolAssetBalance', UniswapPoolAssetBalance);
```

Comment:
we need to bump the version number somewhere too right?

Reply:
in the premium? yes

Reply:
@kelsos only this line I need to update right? https://github.com/rotki/rotki/blob/b94eefa9b16a002b6f901d53dd15a4b4671fd91b/frontend/app/src/premium/setup-interface.ts#L108

Reply:
yes yes :+1: